webpack.config.coffee

```diff
@@ -0,0 +1,34 @@
+ExtractTextPlugin = require 'extract-text-webpack-plugin'
+
+module.exports =
+  entry: "./index.coffee"
+  output:
+    path: 'dist'
+    filename: 'tutor.js'
+    pubicPath: '/dist/'
+
+  plugins: [
+    new ExtractTextPlugin("tutor.css")
+  ]
+
+  module:
+    loaders: [
+      { test: /\.json$/,   loader: "json-loader"       }
+      { test: /\.coffee$/, loader: "coffee-loader"     }
+      { test: /\.cjsx$/,   loader: "coffee-jsx-loader" }
+
+      { test: /\.less$/,  loader: ExtractTextPlugin.extract('css!less') }
+
+      { test: /\.(png|jpg|svg)/, loader: 'file-loader?name=[name].[ext]'}
+      { test: /\.(woff|woff2|eot|ttf)/, loader: "url-loader?limit=30000&name=[name]-[hash].[ext]" }
+   ]
+  resolve:
```

Comment:
We might consider adjusting the limit here, it controls what gets embedded into the stylesheet.  Right now it's embedding the fonts directly.

We could also use the url-loader for the images if we wanted to include them?